### ISSUE TITLE
Get data from Cask

### ISSUE THREAD

**Issue by DamienCassou** [purcell/package-lint#73]
I currently have this in my [Makefile](https://github.com/DamienCassou/klassified.el/blob/master/Makefile) to run package-lint:

```makefile
lint : $(SRCS) clean-elc
	# Run package-lint to check for packaging mistakes
	${CASK} emacs $(EMACSFLAGS) \
	--eval "(require 'package)" \
	--eval "(push '(\"melpa\" . \"http://melpa.org/packages/\") package-archives)" \
	--eval "(package-initialize)" \
	--eval "(package-refresh-contents)" \
	-l package-lint.el \
	-f package-lint-batch-and-exit klassified.el

```

I see several problems here and would like to simplify that:

1. I don't want to specify `package-archives` here as I already specify it in my Cask file
    ```lisp
   (source gnu)
   (source melpa)
   ```
1. I don't want to run `package-initialize` and `package-refresh-contents` manually. This should be the responsibility of the package-lint function I call (e.g., `package-lint-batch-and-exit`).

Basically, I would like to just write:

```Makefile
lint : $(SRCS) clean-elc
	# Run package-lint to check for packaging mistakes
	${CASK} emacs $(EMACSFLAGS) \
	-l package-lint.el \
	-f package-lint-FROM-CASK-batch-and-exit klassified.el
```

Please note the `FROM-CASK` above to tell `package-lint` to use the Cask specification. If you think package-lint is not appropriate for that, what do you think about a `package-lint-cask` responsible for calling `package-lint` properly (in the same way we have `flycheck-cask`).

**Comment by DamienCassou** [comment]
Any insight?

**Comment by purcell** [comment]
I'd be open to adding this to `package-lint`. I'm not sure I'd want to hard-code the list of `package-archives`, though, so it'd still be necessary to pass that in somehow.

**Comment by DamienCassou** [comment]
I don't plan to work on this anymore as I'm using Cask less and less these days. You may close the ticket.

**Comment by bradyt** [comment]
Um.
